src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -66,13 +66,8 @@ public function dispatchEvent($eventName, EventArgs $eventArgs = null): void
     /**
      * @return object[][]
```

Comment:
I suggest removing this phpdoc, so that we inherit the more precise one defined in the parent class (should probably be done in 6.4 btw), especially given that this is a wrong phpdoc as this method does not return nested arrays when an event name is passed.

Reply:
Alright thanks!